requests/packages.py

```diff
@@ -3,8 +3,17 @@
 # This code exists for backwards compatibility reasons.
 # I don't like it either. Just look the other way. :)
 
-for package in ('urllib3', 'idna', 'chardet'):
-    locals()[package] = __import__(package)
+for _package in ('urllib3', 'idna', ('cchardet', 'chardet')):
+    if isinstance(_package, tuple):
+        package = _package[1]
+        try:
+            locals()[package] = __import__(package[0])
```

Comment:
Doesn't look like this line is correct, should be `__import__(_package[0])`? Might want to unpack the tuple right away and then use proper variable names so the logic is easier to follow.